conf/broker.conf

```diff
@@ -764,6 +764,9 @@ saslJaasClientAllowedIds=
 # Default value `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "Broker".
 saslJaasServerSectionName=
 
+# Secret for SaslRoleTokenSigner.
+# Default value `SaslConstants.JAAS_DEFAULT_ROLE_TOKEN_SIGNER_SECRET`, which is "PulsarSecret".
+saslJaasServerRoleTokenSignerSecret=PulsarSecret
```

Comment:
Why don't we leave this blank and force the user to configure it? I agree with @eolivelli that a default secret seems like a security hole.

Also, can we supply the secret as a file? Finally, note that the python script that configures these `.conf` files only obfuscates configs with `password` in the name. As it is written, this would not be obfuscated.

Reply:
ok, I will  supply the secret as a file